packages/scandipwa/src/component/CurrencySwitcher/CurrencySwitcher.container.tsx

```diff
@@ -55,6 +57,7 @@ export class CurrencySwitcherContainer extends DataContainer<CurrencySwitcherCon
 
         try {
             await updateCurrency({ currencyCode });
+            updateCurrentCurrency(currencyCode);
```

Comment:
You forgot to decompose `updateCurrentCurrency` from `props`.

Reply:
agreed, `updateCurrentCurrency` should be from props